CMSgov/cms-carts-seds
# [MDCT-298] You put the kafka in, you put the kafka out

# Description

Lots of fun stuff going on here, following a few pre-established patterns. 

* Set up the bigmac-streams service in the same way v2 is setup. This lets us tie in to inbound kafka events from SEDS via the configureConnectors function and sinkEnrollmentCounts function.
* Tied in the database streams into the postKafkaData function for the tables we care about. This is just state-status, and sections. These get shuttled over to bigmac, where they can be read by dataconnect.

## How to test

Deployed branch:
https://d3a4wxyxbpvu8q.cloudfront.net/

Inbound events:
When events come over the wire, they get added to stg-enrollment-counts. The table `mdct-298-enroll-ok-stg-enrollment-counts` should be populated from playing catchup on deploy.

Outbound events:
You can check the logs for postKafkaData function, and that it doesn't contain errors. Already verified with folks over on the bigmac team that messages are appearing and correctly formatted from this branch.

## Dependencies

None

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
